lib/src/document.dart

```diff
@@ -13,7 +13,7 @@ class Document {
   final ExtensionSet extensionSet;
   final Resolver linkResolver;
   final Resolver imageLinkResolver;
-  final bool encodeHtml;
+  final bool encodeHtml, checkable;
```

Comment:
The checkboxes feature should only be enabled via a separate Syntax, added by default in the `gitHubWeb` ExtensionSet.